traffic_ops/traffic_ops_golang/api/api_route.go

```diff
@@ -0,0 +1,148 @@
+package api
```

Comment:
Is it really a problem if the underscore isn't a prefix (or suffix)? Not that I really care, it could just as easily be `api-route.go`. Also, I'm looking at these from bottom to top, so I missed this before responding to the other one. Looks like we had the same idea.

Reply:
>Is it really a problem if the underscore isn't a prefix (or suffix)?
Not sure what you mean. The suffix is `_route`. So, if Go introduced a `_route.go` tool or directive, this would break that. Unlikely, but trivial to avoid the risk beforehand.